src/perf/lib/Tcp.cpp

```diff
@@ -199,9 +199,19 @@ TcpWorker::~TcpWorker()
     CxPlatEventUninitialize(WakeEvent);
 }
 
-bool TcpWorker::Initialize(TcpEngine* _Engine)
+bool TcpWorker::Initialize(TcpEngine* _Engine, uint16_t AssignedCPU)
 {
     Engine = _Engine;
+    ExecutionContext.Callback = DoWork;
+    ExecutionContext.Context = this;
+    ExecutionContext.Ready = TRUE;
+
+    if (Engine->TcpExecutionProfile == TCP_EXECUTION_PROFILE_LOW_LATENCY && AssignedCPU != UINT16_MAX) {
+        // CxPlatAddExecutionContext(&ExecutionContext, AssignedCPU); TODO - This code does not work.
```

Comment:
After setting the exec context, the CxPlat workers will execute the doWork() function once then seem to start to hang.

Reply:
This works now, but it's unclear why modifying `noWorkCount` and `nextTimeUs = 0` makes this all work.